### DISCUSSION TITLE
iOS and Android with Qt6

### DISCUSSION THREAD

**Discussion by niqt** [zxing-cpp/zxing-cpp#604] [General]
Hi,
is this library working on Android and iOS with Qt6? I have a lot of problems, I'd like to understand if it depends on me or others. Thanks

**Comment by emericg** [comment]
Hi, here is a full demo app for Qt6 (tested with 6.5.3 and 6.6.1) for iOS and Android.

https://github.com/emericg/QmlMobileScanner

It has backends for qzxing, zxingcpp and zint. The qzxing has been modified to work on Qt6, the zxingcpp and zint backends are "new" but inspired from existing code.

**Reply by niqt** [reply]
Thanks, I'll test it!
